sdk/turing/batch/job.py

```diff
@@ -0,0 +1,146 @@
+from enum import Enum
+from typing import List, Optional
+import turing.generated.models
+from .config import EnsemblingJobConfig
+from turing._base_types import ApiObject, ApiObjectSpec
+
+
+class EnsemblingJobStatus(Enum):
+    """
+    Status of ensembling job
+    Possible statuses:
+
+    JobPending --▶ JobFailedSubmission
+        |
+        |
+        |
+        |
+    JobBuildingImage --▶ JobFailedBuildImage
+        |
+        |
+        |
+        |
+        ▼
+    JobRunning --▶ JobFailed
+        |
+        |
+        |--▶ JobTerminated
```

Comment:
There's a JobTerminating process before this.

Reply:
Right